src/modules/75-cet/RouteDestinations.tsx

```diff
@@ -102,103 +102,111 @@ const ETRoutes: FC = () => {
 
       <RouteWithLayout
         licenseRedirectData={licenseRedirectData}
-        sidebarProps={ETSideNavProps}
-        path={routes.toETHome({ ...projectPathProps })}
+        sidebarProps={CETSideNavProps}
+        path={routes.toCETHome({ ...projectPathProps })}
         exact
-        pageName={PAGE_NAME.ETHomePage}
+        pageName={PAGE_NAME.CETHomePage}
       >
-        <ETHomePage />
+        <CETHomePage />
       </RouteWithLayout>
 
       <RouteWithLayout
         layout={MinimalLayout}
         path={routes.toModuleTrialHome({ ...accountPathProps, module: ModuleName.CET.toLowerCase() as Module })}
         exact
-        pageName={PAGE_NAME.ETTrialPage}
+        pageName={PAGE_NAME.CETTrialPage}
       >
-        <ETTrialPage />
+        <CETTrialPage />
       </RouteWithLayout>
 
       <RouteWithLayout
         layout={MinimalLayout}
-        path={routes.toETHomeTrial({ ...accountPathProps })}
+        path={routes.toCETHomeTrial({ ...accountPathProps })}
         exact
-        pageName={PAGE_NAME.ETTrialPage}
+        pageName={PAGE_NAME.CETTrialPage}
       >
-        <ETTrialPage />
+        <CETTrialPage />
       </RouteWithLayout>
 
       <RouteWithLayout
         licenseRedirectData={licenseRedirectData}
-        sidebarProps={ETSideNavProps}
-        path={routes.toETEventsSummary({ ...accountPathProps, ...projectPathProps, ...etModuleParams })}
+        sidebarProps={CETSideNavProps}
+        path={routes.toCETEventsSummary({ ...accountPathProps, ...projectPathProps, ...etModuleParams })}
       >
-        <ETEventsSummary />
+        <CETEventsSummary />
+      </RouteWithLayout>
+
+      <RouteWithLayout
+        licenseRedirectData={licenseRedirectData}
+        sidebarProps={CETSideNavProps}
+        path={routes.toCETEventSummaryOldNotifLink({ ...accountPathProps, ...projectPathProps, ...etModuleParams })}
+      >
+        <RedirectToCETProject />
       </RouteWithLayout>
 
       <RouteWithLayout
         licenseRedirectData={licenseRedirectData}
         exact
-        sidebarProps={ETSideNavProps}
-        path={[routes.toETSettings({ ...accountPathProps, ...projectPathProps, ...etModuleParams })]}
+        sidebarProps={CETSideNavProps}
+        path={[routes.toCETSettings({ ...accountPathProps, ...projectPathProps, ...etModuleParams })]}
       >
-        <RedirectToETControl />
+        <RedirectToCETControl />
       </RouteWithLayout>
 
       <RouteWithLayout
         licenseRedirectData={licenseRedirectData}
         exact
-        sidebarProps={ETSideNavProps}
-        path={[routes.toETAgents({ ...accountPathProps, ...projectPathProps, ...etModuleParams })]}
+        sidebarProps={CETSideNavProps}
+        path={[routes.toCETAgents({ ...accountPathProps, ...projectPathProps, ...etModuleParams })]}
       >
-        <ETSettings>
-          <ETAgents pathComponentLocation={'/agents'} />
-        </ETSettings>
+        <CETSettings>
+          <CETAgents pathComponentLocation={'/agents'} />
```

Comment:
Not a blocker, but no need for the brackets
```suggestion
          <CETAgents pathComponentLocation="/agents" />
```